404-server/OWNERS

```diff
@@ -0,0 +1,4 @@
+assignees:
+- wernight
```

Comment:
@foxish 
Replace with bprashanth. Why didn't findowners choose him?

He wrote the code:
https://github.com/kubernetes/contrib/blame/master/404-server/server.go